### DISCUSSION TITLE
Add conditions to`Horizontal Rule` and `Line Break`

### DISCUSSION THREAD

**Discussion by thupsi** [craftcms/cms#13169] [Ideas] [answered]
I think it would make sense to add conditions to `Horizontal Rule` and `Line Break`. All other UI elements have them, so I see no reason to ommit this functionality for these two elements.

Use case: 
In some layout fields with conditions it's possible that all fields around and between certain UI elements could become hidden. In such cases the "orphaned" UI elements serve no puprose and just look weird.

**Comment by turnstylerj** [comment]
Would really like to see this as well!

**Comment by brandonkelly** [comment, marked as answer]
Added for 5.7! (c7cde6fd07149de16c13d8775c2228bdcff20cfb)

**Reply by brandonkelly** [reply]
Craft 5.7.0 is out with that change.
